ice_melt.dsc

```diff
@@ -0,0 +1,28 @@
+ice_melt:
+  type: world
+  events:
+    on arrow hits *ice in:area_flagged:ice_melt:
```

Comment:
I noticed that ice blocks melt naturally from the light blocks we place near them, can you add an event to prevent them melting in this area?

Reply:
added `block fades` event